test/datalog_tests/simple.ast.expected

```diff
@@ -811,4 +817,19 @@ Cast_u256(.description="264'hfffffffffffffffffffffffffffffffffffffffffffffffffff
 Cast_u256(.description="128'hffffffffffffffffffff as bit<256>", .expected=(128'd1208925819614629174706175 as bit<256>), .actual=256'd1208925819614629174706175).
 Cast_u256(.description="(100: signed<32>) as bit<32> as bit<256>", .expected=(((32'sd100: signed<32>) as bit<32>) as bit<256>), .actual=256'd100).
 Cast_u256(.description="(32'sd0 - 32'sd10) as bit<32> as bit<256>", .expected=(((32'sd0 - 32'sd10) as bit<32>) as bit<256>), .actual=256'd4294967286).
+Cast_bigint(.description="24'd100  as bigint", .expected=(24'd100 as bigint), .actual=100).
+Cast_bigint(.description="32'd100  as bigint", .expected=(32'd100 as bigint), .actual=100).
+Cast_bigint(.description="16'd100  as bigint", .expected=(16'd100 as bigint), .actual=100).
+Cast_bigint(.description="8'd100   as bigint", .expected=(8'd100 as bigint), .actual=100).
+Cast_bigint(.description="128'd100 as bigint", .expected=(128'd100 as bigint), .actual=100).
+Cast_bigint(.description="256'd100 as bigint", .expected=(256'd100 as bigint), .actual=100).
+Cast_bigint(.description="512'd100 as bigint", .expected=(512'd100 as bigint), .actual=100).
+Cast_bigint(.description="264'hfffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffa as bigint", .expected=(264'd29642774844752946028434172162224104410437116074403984394101141506025761187823610 as bigint), .actual=29642774844752946028434172162224104410437116074403984394101141506025761187823610).
+Cast_bigint(.description="(32'sd0 - 32'sd10) as bigint", .expected=((32'sd0 - 32'sd10) as bigint), .actual=(0 - 10)).
+UMinus_s32(.description="-32'sd100", .n=(- 32'sd100)).
```

Comment:
printing these with a space is a bit confusing

Reply:
Now you're just nitpicking :)